### PR TITLE
Update NuGetCommand.resx

### LINKED ISSUE #9122: Doc: "nuget config /?" has typo in config get exapmle
## Details about Problem

NuGet product used NuGet.exe | VS UI | Package Manager Console | dotnet.exe): NuGet.exe

NuGet version (x.x.x.xxx): 5.4.0.6315

## Detailed repro steps so we can see the same problem

1. Run `nuget config /?`:

  ```
  usage: NuGet config <-Set name=value | name>

  Gets or sets NuGet config values.

  options:

   -Set +                        One on more key-value pairs to be set in config.
   -AsPath                       Returns the config value as a path. This option is ignored when -Set is specified.
   -Help                    (?)  help
   -Verbosity                    Display this amount of details in the output: normal, quiet, detailed.
   -NonInteractive               Do not prompt for user input or confirmations.
   -ConfigFile                   The NuGet configuration file. If specified, only the settings from this file will be used. If not specified, the hierarchy of configuration files from the current directory will be used. To learn more abou
                                 t NuGet configuration go to https://docs.microsoft.com/en-us/nuget/consume-packages/configuring-nuget-behavior.
   -ForceEnglishOutput           Forces the application to run using an invariant, English-based culture.

  examples:

  nuget config -Set HTTP_PROXY=http://127.0.0.1 -Set HTTP_PROXY.USER=domain\user
  nuget.config HTTP_PROXY

  For more information, visit https://docs.nuget.org/docs/reference/command-line-reference
  ```

2. See that the `nuget config HTTP_PROXY` command has an extra dot.

### PR DESCRIPTION
**Pull request by mahilleb-msft**
## Bug

Fixes: NuGet/Home#9122
Regression: Unknown
* Last working version:
* How are we preventing it in future:

## Fix

Details: fix the typo

## Testing/Validation

Tests Added: No
Reason for not adding tests: simple typo
Validation:

### CODE REVIEW

**Comment by zivkan** [comment]
@mahilleb-msft please keep our PR template, our tools use it for release notes and similar things. This includes creating an issue in NuGet/Home to link to.

File: src/NuGet.Clients/NuGet.CommandLine/NuGetCommand.resx [hunk 1]
```diff
@@ -122,7 +122,7 @@
   </data>
   <data name="ConfigCommandExamples" xml:space="preserve">
     <value>nuget config -Set HTTP_PROXY=http://127.0.0.1 -Set HTTP_PROXY.USER=domain\user
-nuget.config HTTP_PROXY</value>
+nuget config HTTP_PROXY</value>
```
Comment (nkolev92) [inline src/NuGet.Clients/NuGet.CommandLine/NuGetCommand.resx:125]: You should make sure the generated file gets changed as well. 

So either do a build or make this change in VS.

**Comment by nkolev92** [comment]
@mahilleb-msft 

Can you update this PR?
Thanks.

**Comment by mahilleb-msft** [comment]
hey @nkolev92 - PR updated.

**Review by zivkan — APPROVED** [review]

**Pull request merged**